sycl/test-e2e/ESIMD/thread_id_test.cpp

```diff
@@ -2,7 +2,7 @@
 // UNSUPPORTED: cuda || hip
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
-// XFAIL: gpu && !esimd_emulator
+// XFAIL: linux && gpu && !esimd_emulator

```

Comment:
Please, do not make changes not related to pulling down community changes.

Reply:
This test is unexpectedly passing on Windows. Should I ignore that failure?

Reply:
Yes. The test unexpectedly passing in sycl branch.

Reply:
Reverted my change.